draft-ietf-quic-tls.md

```diff
@@ -805,10 +805,12 @@ The HKDF-Expand-Label function defined in TLS 1.3 MUST be used for Initial
 packets even where the TLS versions offered do not include TLS 1.3.
 
 The secrets used for protecting Initial packets do not change during the
-connection, even though the destination connection ID in client Initial packets
-changes after receiving a Retry.  A server that sends a Retry
-therefore needs to either remember the original connection ID
-or save the original connection ID in the Retry token.
+connection, even though the destination connection ID in client Initial
+packets changes after receiving a Retry.  A server that sends a Retry
+therefore needs to either remember the original connection ID or save
+the original connection ID in the Retry token.  Because the initial
+connection ID is included in the server's transport parameters, the only
+difference is when in the packet processing this connection ID is utilized.
```

Comment:
This reads like justification for a change, rather than static text.  How about:

> The initial connection ID is needed by a server to reconstruct packet protection keys and so that it can produce the correct value for the original_connection_id transport parameter.

And are we using "initial" or "original" here?  I went with original on the basis that it distinguished this from Initial packets, but it's not clear to be that that distinction is necessary, or even a good idea.  After all, this connection ID is what is driving packet protection for those packets; the linkage might be useful.

Reply:
At the time, we were trying to avoid confusion between the _original_ DCID, which generated the keys, and the DCID in the current Initial.  It was a useful distinction in the old design.  With this change, I think that distinction is no longer useful.